cfme/tests/configure/test_workers.py

```diff
@@ -45,9 +63,67 @@ def test_restart_workers(appliance):
              message="Wait for all original workers are back online")
 
 
+v510 = {
+    "1.1.gigabytes": 1178599424,
+    1178599424: "1.1.gigabytes",
+    "1.2.gigabytes": 1283457024,
+    1283457024: "1.2.gigabytes"
+}
+
+
+@pytest.fixture(scope="module")
+def _view(appliance):
```

Comment:
Why is this fixture starting with an underscore?

I see your note about the impact to execution time, but I don't think its appropriate to have this as a fixture, as there's no setup, just navigation that has to happen in the test anyway.

The navigation framework should minimize the navigation necessary throughout the tests.

If the browser is already on the right page, a call to navigate_to should not force naviagation by default, it should be using the `is_displayed` method on the view class or `am_i_here` method on the navigation class to prevent unnecessary navigation.  Have you investigated that behavior within the test to make sure it isn't re-navigating when it shouldn't? That should be fixed in the navigation/view classes, not via fixture scoping in the test.

Reply:
I started this fixture with an underscore to indicate that it's used just locally.
It is not re-navigating, am_i_here in this case just takes really a lot of time. I didn't check what am_i_here does, but it took half of test time before I started skipping the check, as I know that I am all the time on the same page.

Reply:
I couldn't find the reason why is the is_displayed/am_i_here check is so slow. I just saw it checks a lot of things and calls other is_displayed functions. I have removed the optimization as you say it's not appropriate to have it as a fixture. However, run time of the tests is doubled (for me ~7 mins with the optimization, ~15 mins without the optimization)

Reply:
I think `as I know that I am all the time on the same page` is an assumption on clean test results, not a known :)

Let's debug through the time taken to validate the page separately from this test automation, thanks for rolling that change back!  It certainly should not have that impact on the test, as the whole intent is to shorten navigation time!

Reply:
I'm also seeing the test_set_memory_threshold tests run in PRT in <9m.

Reply:
Good point that it's just an assumption. :+1:  And if there are not many tests (or tests to do) like that (with slow is_displayed), then I guess it's also not the priority to optimize the speed.